syntax = "proto3";
package heartBeat;
//option py_generic_services = true;
option cc_generic_services = true;
// option java_generic_services = true;
// option java_package = "echo";
// option java_outer_classname = "EchoProto";


message HeartBeatRequest {
  string heartBeartReq = 1;
}

message HeartBeatResponse {
  string heartBeartRsp = 2;
}

service HeartBeatService {
  rpc HeartBeat(HeartBeatRequest) returns (HeartBeatResponse);
}

